server/src/render.js

```diff
@@ -3,21 +3,44 @@ import path from "node:path";
 import os from "node:os";
 import { chromium } from "playwright";
 import pino from "pino";
+import mhtml2html from "mhtml2html";
+import { JSDOM } from "jsdom";
 
 const logger = pino({
   level: process.env.LOG_LEVEL || "info",
 });
 
-export async function render(actualHtml, viewport, fullpage, style) {
+export async function render(
+  actualMhtml,
+  viewport,
+  fullpage,
+  style,
+  usehMhtmlConverter,
+) {
   const renderer = new BrowserRenderer();
   const workspaceDirectory = await fs.mkdtemp(
     path.join(os.tmpdir(), "pixelpact-"),
   );
-  const indexFile = `${workspaceDirectory}/index.mhtml`;
-  await fs.writeFile(indexFile, actualHtml);
+  let indexFile;
+  if (usehMhtmlConverter) {
+    indexFile = `${workspaceDirectory}/index.html`;
+    const convertedHtml = mhtml2html.convert(actualMhtml, {
+      parseDOM: (html) => new JSDOM(html),
+    });
```

Comment:
As we've already discussed, neither of us really likes this dependency. But at this stage of the project, it's about doing the right thing first, and then doing it right/fast in a second step.